class/defaults.yml

```diff
@@ -90,7 +90,7 @@ parameters:
           hard:
             requests.cpu: 1000m
```

Comment:
might make sense to bump CPU requests also then:

```suggestion
            requests.cpu: 4
```

Reply:
makes sense, yes